contract/src/main/java/io/wax911/emojify/contract/serializer/IEmojiDeserializer.kt

```diff
@@ -0,0 +1,8 @@
+package io.wax911.emojify.contract.serializer
+
+import io.wax911.emojify.contract.model.AbstractEmoji
+import java.io.InputStream
+
+interface IEmojiDeserializer {
+    fun decodeFromStream(inputStream: InputStream): List<AbstractEmoji>
```

Comment:
If we move the remaining properties into `IEmoji` we probably want to substitute with the interface instead. (Applies to all the cases below)
```suggestion
    fun decodeFromStream(inputStream: InputStream): List<IEmoji>
```

Reply:
`IEmoji` is an internal interface, wouldn't it be better to keep `AbstractEmoji` so that `unicode`, `htmlDec` and `htmlHex` are implemented the right way ?

Reply:
That's fair, I'm happy with this